pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -631,8 +631,9 @@ func osdOnSDNFlag(hostnetwork bool, v cephver.CephVersion) []string {
 	// for more details: https://github.com/rook/rook/issues/3140
 	if !hostnetwork {
 		if v.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 2}) {
-			args = append(args, "--ms-learn-addr-from-peer", "false")
+			args = append(args, fmt.Sprint("--ms-learn-addr-from-peer=false"))
```

Comment:
Is the `fmt.Sprint` needed here if nothing is "templated" here?

Reply:
Not sure to understand what you mean here, somehow that's the only way I've managed to have it working (as per the commit message).

Reply:
I mean the `fmt.Sprint(` around it.

Instead of
```
args = append(args, fmt.Sprint("--ms-learn-addr-from-peer=false"))
```
to
```
args = append(args, "--ms-learn-addr-from-peer=false")
```

Reply:
Ahhh I see, yes that should work as well. Let me correct that.